DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/part3_ddr_hosts/src/mmhost.cpp

```diff
@@ -0,0 +1,118 @@
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+#include <sycl/ext/oneapi/annotated_arg/annotated_ptr.hpp>
+
+#include "exception_handler.hpp"
+
+using namespace sycl;
+using namespace ext::oneapi::experimental;
+
+using usm_buffer_location =
+    ext::intel::experimental::property::usm::buffer_location;
+
+constexpr int kBL1 = 1;
+constexpr int kBL2 = 2;
+
+struct DDR_IP{
+  using params = decltype(properties{
+          buffer_location<kBL1>,
+          maxburst<8>,
+          dwidth<256>,
```

Comment:
HLS tutorial also specified `latency<0>`. We haven't switched the default to latency<0> yet.

Reply:
I see, latency default is then 1 as per the docs https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_intel_fpga_kernel_arg_properties.asciidoc
correct?